apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -111,6 +127,51 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
     setAdvancedSettingsOpen(!advancedSettingsOpen);
   };
 
+  const recordSectionSetupEvent = (e, section) => {
+    e.preventDefault();
+    const initial = initialSection.current;
```

Comment:
nit: I know there's some weirdness here with variable names but could we call this `initialSection` and rename `initialSection` to `initialSectionRef` or something? It took me a while to figure out what this object represented (it doesn't help that `current` also meant nothing to me, though I understand that's a React thing)

Reply:
Good call- updated! And ugh yeah .current is a very weird React label to me... especially when I don't really want 'current' in reality- I want 'initial'.